### PR TITLE
Flux v2.2.0 release blog

### PR DESCRIPTION
**Pull request by hiddeco**
Supersedes: #1733
Part of: https://github.com/fluxcd/flux2/issues/4410

:eyes: **Preview:** https://deploy-preview-1744--fluxcd.netlify.app/blog/2023/12/flux-v2.2.0/

### CODE REVIEW

File: content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+author: hiddeco
+date: 2023-12-12 16:00:00+00:00
+title: Announcing Flux 2.2 GA
+description: "Flux v2.2.0: Improving your Helm experience!"
```
Comment (hiddeco) [inline content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md:5]: Suggestions are welcome, as long as it's within bounds of not being too enthusiastic.

File: content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+- `.spec.matchOIDCIdentity[].issuer` allows you to specify a regex pattern to match against the issuer of the certificate related to the artifact signature used for OCI keyless verification.
+- `.spec.matchOIDCIdentity[].subject` allows you to specify a regex pattern to match against the subject of the certificate related to the artifact signature used for OCI keyless verification.
+
+### Bucket v1beta2
+
+- A new field, `.spec.prefix`, has been added which enables server-side filtering of files if the object's .spec.provider is set to generic/aws/gcp.
+
+### HelmRepository v1beta2 and ImageRepository v1beta2
+
+- A new boolean field, `.spec.insecure`, has been introduced which allows connecting to a non-TLS HTTP container registry. For HelmRepository API, it is only considered if the object's `.spec.type` is set to `oci`.
```
Comment (hiddeco) [inline content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md:36]: Lists do not work nicely when the lists are short and you want to maintain a reader their attention. Should rewrite this to be fluent.

File: content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+## Breaking changes to Kustomizations
+
+All Flux components have been updated from Kustomize v5.0.3 to [v5.3.0](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv5.3.0). 
+
+You should be aware that this update comes with a breaking change in Kustomize, as components are now applied after generators. If you use Kustomize components or `.spec.components` in Kustomizations along with generators, then please make necessary changes before upgrading to avoid any undesirable behavior. For more information, see the relevant [Kustomize issue](https://github.com/kubernetes-sigs/kustomize/issues/5141).
+
+## Other notable changes
+
+- `flux install` has been made safer by preventing unwanted upgrades to a bootstrapped cluster.
+- `flux bootstrap` now supports Gitea. To bootstrap Flux onto a cluster using Gitea as the Git provider, run `flux bootstrap gitea --repository <repo> --owner <owner>`.
```
Comment (hiddeco) [inline content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md:149]: Some type of section to refer to the upgrade guide?

File: content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
+---
+author: hiddeco
+date: 2023-12-12 16:00:00+00:00
```
Comment (stefanprodan) [inline content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md:3]: Let's set this at 14

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @hiddeco 🏅

File: content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md (hunk 4 above)
Comment (hiddeco) [reply content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md:3]: I think 15 should be fine, given it's UTC.

**Pull request merged**